### PR TITLE
Fixed: Clarify IPT cookie help text

### PR DESCRIPTION
**Pull request by PearsonFlyer**
#### Database Migration
NO

#### Description
The cookie helptext for IPT was confusing users, so clarified it with a reference to the wiki where detailed instructions are available on how to find the cookie required for this field.

#### Screenshot (if UI related)
image

#### Todos
- [ ] Tests
- [ ] Translation Keys (./src/NzbDrone.Core/Localization/Core/en.json)
- [ ] [Wiki Updates](https://wiki.servarr.com)

#### Issues Fixed or Closed by this PR

* Fixes #XXXX

### CODE REVIEW

**Comment by bakerboy448** [comment]
Needs a rebase on (upstream) develop; think you forgot to pull upstream develop when you created the branch 

this is also a GitHub bot that will keep your forked develop in sync with upstream.

do we have a wiki link specifically addressing IPT?  

The Reddit post was two issues 1) how to get the cookie (this is covered on the faq) and 2) what information from the cookie needs to be input and how.

**Review by bakerboy448 — CHANGES_REQUESTED** [review]
See other comment

**Comment by bakerboy448** [comment]
Needs a rebase on (upstream) develop; think you forgot to pull upstream develop when you created the branch 

this is also a GitHub bot that will keep your forked develop in sync with upstream.

do we have a wiki link specifically addressing IPT?  

The Reddit post was two issues 1) how to get the cookie (this is covered on the faq) and 2) what information from the cookie needs to be input and how.

File: src/NzbDrone.Core/Indexers/Definitions/IPTorrents.cs [hunk 1]
```diff
@@ -364,7 +364,7 @@ public IPTorrentsSettings()
         [FieldDefinition(1, Label = "Base Url", Type = FieldType.Select, SelectOptionsProviderAction = "getUrls", HelpText = "Select which baseurl Prowlarr will use for requests to the site")]
         public string BaseUrl { get; set; }
 
-        [FieldDefinition(2, Label = "Cookie", HelpText = "Site Cookie")]
+        [FieldDefinition(2, Label = "Cookie", HelpText = "Site Cookie - see wiki for more info")]
```
Comment (Qstick) [inline src/NzbDrone.Core/Indexers/Definitions/IPTorrents.cs:367]: If we have a link, use “HelpLink” prop so that we link to it

Comment (bakerboy448) [inline src/NzbDrone.Core/Indexers/Definitions/IPTorrents.cs:367]:
```suggestion
        [FieldDefinition(2, Label = "Cookie", HelpText = "Enter the cookie for the site. Example abbreviated cookie formatting: `PHPSESSID=8rk27odm; ipsconnect_63ad9c=1; more_stuff=etc;`", HelpLink = "https://wiki.servarr.com/prowlarr/faq#finding-cookies")]
```

Comment (bakerboy448) [reply src/NzbDrone.Core/Indexers/Definitions/IPTorrents.cs:367]: suggested code work?
image

is it possible to do a code block for the cookie formatting?

Comment (bakerboy448) [reply src/NzbDrone.Core/Indexers/Definitions/IPTorrents.cs:367]: updated; thanks to Nit for the cookie example

**Comment by bakerboy448** [comment]
conflicts need resolving as well

**Review by bakerboy448 — APPROVED** [review]

**Pull request merged**